examples/plugins/project.json

```diff
@@ -4,37 +4,18 @@
   "sourceRoot": "examples/plugins/src",
   "projectType": "library",
   "targets": {
-    "build": {
-      "executor": "@nx/js:tsc",
-      "outputs": ["{options.outputPath}"],
-      "options": {
-        "outputPath": "dist/examples/plugins",
-        "main": "examples/plugins/src/index.ts",
-        "tsConfig": "examples/plugins/tsconfig.lib.json",
-        "assets": ["examples/plugins/*.md"]
-      }
-    },
-    "lint": {
-      "executor": "@nx/linter:eslint",
-      "outputs": ["{options.outputFile}"],
-      "options": {
-        "lintFilePatterns": ["examples/plugins/**/*.ts"]
-      }
-    },
-    "unit-test": {
-      "executor": "@nx/vite:test",
-      "outputs": ["{options.reportsDirectory}"],
-      "options": {
-        "configFile": "examples/plugins/vitest.unit.config.ts",
-        "reportsDirectory": "../../coverage/examples-plugins/unit-tests"
-      }
-    },
+    "build": {},
+    "lint": {},
+    "unit-test": {},
     "int-test": {
+      "cache": true,
+      "outputs": ["{workspaceRoot}/coverage/{projectName}/int-tests/lcov.info"],
       "executor": "@nx/vite:test",
-      "outputs": ["{options.reportsDirectory}"],
       "options": {
-        "configFile": "examples/plugins/vitest.int.config.ts",
-        "reportsDirectory": "../../coverage/examples-plugins/int-tests"
+        "configFile": "{projectRoot}/vitest.int.config.ts",
+        "coverage": {
+          "enabled": true
+        }
       }
```

Comment:
The configuration covered by `nx.json`, I believe. So the `project.json` could be simplified to `"int-test": {}`.

Same goes for all other projects, as far as I can tell. It looks like you only reduced the `unit-test` configs, but not the `int-test` configs.

Reply:
Done